contracts/pool/PoolFactory.sol

```diff
@@ -10,15 +10,20 @@ import "../interfaces/IPoolFactory.sol";
 
```

Comment:
Is blended pool created via PoolFactory?
How have you been deploying blended pool? We can't find anything related to deployments.